frontend/dashboard/pages/Dashboard/Dashboard.tsx

```diff
@@ -71,7 +70,7 @@ export const Dashboard = ({ user, organizations, disableDebounce }: DashboardPro
               )}
             </div>
             <Link
-              to={'/' + selectedContext + '/new'}
+              href={'/dashboard/' + selectedContext + '/new'}
```

Comment:
After remove `Link `which imported from `react-dom` and replaced `to `with `href `the link did now worked until i added `dashboard`, not  sure if it good solution🤔

Reply:
I don't see any problem with this😊
We also have Playwright tests that will fail if there is anything wrong with this link.